src/engine-actions/pay-invoice.js

```diff
@@ -6,12 +6,11 @@ const {
  * Given a payment request, it pays the invoices and returns a refund invoice
  *
  * @param {String} paymentRequest
- * @param {Object} options
- * @param {Number} expiry expiration of refund invoices
- * @return {String} paymentPreimage
+ * @return {Promise<string>} paymentPreimage
  */
 
-async function payInvoice (paymentRequest, options = {}) {
+async function payInvoice (paymentRequest) {
+  // @ts-ignore
```

Comment:
why the ignore here?

Reply:
It appears to be an invalid usage of `sendPaymentSync` according to the LND documentation, though I'm guessing the documentation is just wrong.

Reply:
I think we should mark this as a TODO to make the parameters for `sendPayment` actually reflect what's required (using either the payment request: https://github.com/lightningnetwork/lightning-rfc/blob/master/11-payment-encoding.md#bolt-11-invoice-protocol-for-lightning-payments or the actual fields)

Reply:
Where should we mark it? Should I make a ticket for enabling strict mode and add this as a note for it?

Reply:
I fixed it so that we no longer need the ignore.